MultiLoggerSingleton2/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace MultiLoggerSingleton2
+{
+    internal class Program
+    {
+        /// <summary>
+        /// This Program is an example of Thread Safe Singleton - MultiLogger with different files.
+        /// </summary>
+        static void Main()
+        {
+            string fileName1 = "D:\\GitHub\\DesignPatterns\\MultiLoggerSingleton\\log1.text";
```

Comment:
text is never any extension.

Either it should be txt or log

Reply:
Updated